catalog/core/catalog-core-migratable/src/test/java/org/codice/ddf/catalog/migratable/impl/MigrationFileWriterTest.java

```diff
@@ -210,6 +180,17 @@ public void testWriteMetacardsWriteObjectFails() throws Exception {
         List<Result> results = new ArrayList<>();
         results.add(new ResultImpl(new MetacardImpl()));
 
+        mockFile = new File("");
```

Comment:
Turns out in this case, a file _was_ getting written out. It will be a left-over after the test. Nice catch @tbatie -- fix will be to use a temp file.